packages/icons/icons.json

```
@@ -1227,91 +1227,91 @@
     {
         "displayName": "Asterisk",
         "iconName": "asterisk",
-        "tags": "note, symbol, starred, marked",
+        "tags": "note, symbol, starred, marked, star",
         "group": "miscellaneous",
         "codepoint": 61740
     },
     {
         "displayName": "Cloud",
         "iconName": "cloud",
-        "tags": "file, storage, weather",
+        "tags": "file, storage, weather, sky",
         "group": "file",
         "codepoint": 61786
     },
     {
         "displayName": "Cloud: download",
         "iconName": "cloud-download",
-        "tags": "file, storage, transfer",
+        "tags": "file, storage, transfer, download, cloud",
```

```suggestion
        "tags": "file, storage, transfer",
```

redundant with icon name